proto/one/common/notification/signalr/signalr_one_hub_event_content_map.proto

```diff
@@ -0,0 +1,8 @@
+syntax = "proto3";
+
+option csharp_namespace ="ONE.Models.CSharp";
+
+message SignalROneHubEventContentMap {
+    map<string, string> content = 1;
```

Comment:
I think we need some better naming currently I think this will lead to something like message.EventInfo.Content.Content.Default.Content

Reply:
Here is an example
```json
Text
{
    "Recipient": {
        "AppId": "App1",
        "ConnectionId": "C1",
        "TenantId": "T1",
        "UserId": "U1"
    },

    "Sender": {
        "AppId": "App2",
        "ConnectionId": "C3",
        "TenantId": "T1",
        "UserId": "U3"
    },
    "EventInfo": {
        "Area": "Dashboard",
        "Name": "Dataset_ready",
        "Time": {
            "JsonDateTime": "2023-04-27",
            "Ticks": null,
            "JsonTicksDateTimeCase": 1
        },
        "Content": {
            "Default": {
                "Content": {
                    "DatasetTwinRefId": "TwinRefId"
                }
            },
            "DatasetEvent": null,
            "ContentCase": 1
        }
    }
}
```

Reply:
So one less content than I thought.  Still not great.
What about something like
EventPayload.MappedEvent.Items OR EventPayload.KeyValueEvent.Items
Now that I think about it why not use the KeyValues protobuf there? It's not really a default since it will just be null if not set.  I think several of the fields could benefit from adding 'event' to the field name.